app/serializers/financial_transaction_serializer.rb

```diff
@@ -0,0 +1,18 @@
+class FinancialTransactionSerializer < ActiveModel::Serializer
+  include ApplicationHelper
+
+  attributes :id, :user_id, :user_name, :amount, :note, :created_at
+
+  def user_name
+    show_user(object.user)
+  end
+
+  def amount
+    object.amount.to_f
+  end
+
+  def created_at
+    # use Rails convention in API, which is less easier to migrate later
```

Comment:
we not rename it in the model?

Reply:
Good one --> #575

Reply:
can't we move this getter already into the model (if you don't want to rename them before this change)?